src/components/Prayer.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+const Prayer = () => {
+  const [city, cityState] = useState("gaza strip");
+  const [searchTerm, setSearchTerm] = useState("");
+  const [data, setData] = useState("");
```

Comment:
I think the default value should be empty array here which will contain prayer times.

Reply:
There are other data we want to get from the response, so we sort all the responses and passed the data we want as a props.